### DISCUSSION TITLE
Looking for Live show reference to "the phoenix project"

### DISCUSSION THREAD

**Discussion by jpconstantineau** [BretFisher/ama#235] [Q&A] [answered]
Need show id where Bret introduced "The phoenix project" and referred to it causing PTSD.

I want to include a link/snippet of that show in one of my recommendations for reading at work... 

Made the mistake almost 2 years ago of not bookmarking the show.  It would have been sometime before March 2023 when I bought the book. 
Googling, searching on YouTube didn't really help find the 5 minutes in that show that I am looking for...

**Comment by BretFisher** [comment, marked as answer]
Found it: https://www.youtube.com/live/NhthvqTx1ww?si=X_jH5aMyCNPo5PFP&t=1093

You can search transcripts by clicking on show description, scroll to transcripts button, which opens the full transcript with clickable timestamps! 

What a memory you've got! Enjoy!

**Reply by jpconstantineau** [reply]
Thanks Bret. I thought it was earlier than that but this will definitely do. 

Thanks again.
